### PR TITLE
First commit of boilerplate

### PR DESCRIPTION
**Pull request by AndrewBestbier**

### CODE REVIEW

File: client/utils/spectreFetch.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/*
```
Comment (garrensmith) [inline client/utils/spectreFetch.js]: Could we rename this file to `ajax.js` and then change the methods from `SpectrePost`, `SpectreGet` etc to just `Post`, `Get`

File: lib/email.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  auth: {
+    user: config.mandrill.user,
+    pass: config.mandrill.password
+  }
+});
+
+export function sendResetPasswordEmail (uniqueIdentifier, email) {
+  const mailOptions = {
+    to: email,
+    from: 'andrew@redcometlabs.com',
```
Comment (garrensmith) [inline lib/email.js]: Could you change this from a hardcoded email to a defined email address in the config.

**Comment by garrensmith** [comment]
Just a few small comments. Otherwise I'm +1

File: package.json [hunk 3]
```diff
@@ -0,0 +1,81 @@
+{
+  "name": "Spectre",
+  "description": "Hapi-React-Couch",
+  "version": "0.0.1",
+  "author": "Garrren Smith <garren.smith@gmail.com>",
```
Comment (garrensmith) [inline package.json]: Add yourself here

Comment (garrensmith) [inline package.json]: Also can you add an MIT license

**Comment by AndrewBestbier** [comment]
+1

**Pull request merged**